docroot/themes/humsci/humsci_basic/patterns/horizontal-card/horizontal-card.html.twig

```diff
@@ -2,48 +2,56 @@
 
 <div{{ attributes }}>
 
-  {% if image %}
-    <div class="horizontal-card__img">
+
+  <div class="horizontal-card__img">
```

Comment:
TODO: I need to make sure a horizontal card w/o an image renders the text to the left